src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -34,7 +34,9 @@ public sealed class ZipkinExporterOptions
         /// Gets or sets the name of the service reporting telemetry. If the `Resource` associated with the telemetry
         /// has "service.name" defined, then it'll be preferred over this option.
         /// </summary>
-        public string ServiceName { get; set; } = DefaultServiceName;
+        public string ServiceName { get; set; } =
```

Comment:
We can simply remove this field from ZipkinExporterOptions.

Reply:
I think the zipkin spec used to say options were required, but I just checked and I think we are safe to remove now: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/zipkin.md#service-name

Reply:
Yes. It was intentionally removed from spec.
https://github.com/open-telemetry/opentelemetry-specification/pull/1386/files#r566248088

Reply:
Sounds good, I'll push its removal.
![image](https://user-images.githubusercontent.com/29290481/106551973-45f8bd00-64cb-11eb-822a-dfb7f85b9902.png)
Are we sure that service name is not commonly set using this though? For instance [Microservices Example](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/MicroserviceExample/WorkerService/Program.cs) uses this in two different spots.

Reply:
Those examples should move to use resource instead.